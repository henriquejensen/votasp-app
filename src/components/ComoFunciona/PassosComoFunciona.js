import React from 'react';

import icone1 from '../../img/icone-01.svg';
import icone2 from '../../img/icone-02.svg';
import icone3 from '../../img/icone-03.svg';
import icone4 from '../../img/icone-04.svg';

export default () => (
  <div className="container">
    <div className="row">
      <div className="col-sm-12 col-md-6 col-lg-3">
        <figure className="passos passo1">
          <img className="icone" src={icone1} alt="Icone do questionário" />
        </figure>
        <p className="bold">Você responde um questionário de (no mínimo) 21 questões.</p>
      </div>
      <div className="col-sm-12 col-md-6 col-lg-3">
        <figure className="passos passo2">
          <img
            className="icone"
            src={icone2}
            alt="Icone do cruzamento das suas respostas com as dos candidatos e candidatas."
          />
        </figure>
        <p>Nós cruzamos as suas respostas com os candidatos e candidatas cadastradas.</p>
      </div>
      <div className="col-sm-12 col-md-6 col-lg-3">
        <figure className="passos passo3">
          <img
            className="icone"
            src={icone3}
            alt="Icone 3 dos candidatos com mais afinidade com você."
          />
        </figure>
        <p>Te informamos quem dos candidatos e candidatas pensam mais parecido com você.</p>
      </div>
      <div className="col-sm-12 col-md-6 col-lg-3">
        <figure className="passos passo4">
          <img
            className="icone"
            src={icone4}
            alt="Icone 4. Você pode olhar as respostas de todos os candidatos e candidatas."
          />
        </figure>
        <p>Você pode olhar as respostas e perfil de todos eles e elas.</p>
      </div>
    </div>
  </div>
);