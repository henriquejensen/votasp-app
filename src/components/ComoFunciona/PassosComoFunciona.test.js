import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import PassosComoFunciona from './PassosComoFunciona';

describe('PassosComoFunciona', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(( <PassosComoFunciona /> )))).toMatchSnapshot();
  })
})
