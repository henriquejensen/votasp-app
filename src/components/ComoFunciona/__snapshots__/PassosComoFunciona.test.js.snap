// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PassosComoFunciona should match a snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12 col-md-6 col-lg-3"
    >
      <figure
        className="passos passo1"
      >
        <img
          alt="Icone do questionário"
          className="icone"
          src="test-file-stub"
        />
      </figure>
      <p
        className="bold"
      >
        Você responde um questionário de (no mínimo) 21 questões.
      </p>
    </div>
    <div
      className="col-sm-12 col-md-6 col-lg-3"
    >
      <figure
        className="passos passo2"
      >
        <img
          alt="Icone do cruzamento das suas respostas com as dos candidatos e candidatas."
          className="icone"
          src="test-file-stub"
        />
      </figure>
      <p>
        Nós cruzamos as suas respostas com os candidatos e candidatas cadastradas.
      </p>
    </div>
    <div
      className="col-sm-12 col-md-6 col-lg-3"
    >
      <figure
        className="passos passo3"
      >
        <img
          alt="Icone 3 dos candidatos com mais afinidade com você."
          className="icone"
          src="test-file-stub"
        />
      </figure>
      <p>
        Te informamos quem dos candidatos e candidatas pensam mais parecido com você.
      </p>
    </div>
    <div
      className="col-sm-12 col-md-6 col-lg-3"
    >
      <figure
        className="passos passo4"
      >
        <img
          alt="Icone 4. Você pode olhar as respostas de todos os candidatos e candidatas."
          className="icone"
          src="test-file-stub"
        />
      </figure>
      <p>
        Você pode olhar as respostas e perfil de todos eles e elas.
      </p>
    </div>
  </div>
</div>
`;
